website/content/docs/concepts/domain-model/auth-methods.mdx

```diff
@@ -31,6 +31,97 @@ The password auth method has the following additional attributes:
 
 - `min_password_length` - (required) The default is 8.
 
+### LDAP Auth Method Attributes
+
+The ldap auth method has the following additional attributes:
+
+- `state` - The state of the auth method; either inactive, active-private or
+  active-public
+
+- `start_tls` - (optional) If true, issues a StartTLS command after establishing
+  an unencrypted connection. Defaults to false.
+
+- `insecure_tls` - (optional) If true, skips LDAP server SSL certificate
+  validation - insecure and use with caution. Defaults to false.
+
+- `discover_dn` - (optional) If true, use anon bind to discover the bind DN of a
```

Comment:
Should this maybe have DN (Distinguished Name) since this is the first time DN is seen on this page?